.laminas-ci.json

```diff
@@ -1,5 +1,10 @@
 {
+    "additional_composer_arguments": [
+        "--no-scripts",
+        "--no-plugins"
+    ],
     "ignore_php_platform_requirements": {
-        "8.1": true
+        "8.0": false,
+        "8.1": false
```

Comment:
I've indicated that the `--ignore-platform-requirement` flag should NOT be used with PHP 8.0 and 8.1. `composer update` operations run normally without the flag, and, when using `--prefer-lowest`, we need to get requirements that are appropriate for those versions (otherwise, tests fail due to syntax errors!).

Reply:
AFAIK, if the property `ignore_php_platform_requirements` is available, all PHP versions will have `false` by default.
So it would be enough to simply create the property as an empty object to ensure that platform reqs won't be ignored.

If the `ignore_php_platform_requirements` property is not available, PHP 8.0 will always have ignored platform reqs.